### PR TITLE
Remove duplicate visualization tests and fixes DYN-5962

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

Remove duplicate WPFVisualizationTests

Also fixes [DYN-5962](https://jira.autodesk.com/browse/DYN-5962)

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes
Remove duplicate visualization tests

### Reviewers
@mjkkirschner

### CODE REVIEW

File: src/VisualizationTests/HelixImageComparisonTests.cs [hunk 1]
```diff
@@ -10,18 +10,26 @@
...
 using DynamoCoreWpfTests.Utility;
 using HelixToolkit.Wpf.SharpDX;
 using NUnit.Framework;
 
 namespace WpfVisualizationTests
 {
-
     [TestFixture]
-    public class HelixImageComparisonTests : HelixWatch3DViewModelTests
+    public class HelixImageComparisonTests : VisualizationTest
```
Comment (mjkkirschner) [inline src/VisualizationTests/HelixImageComparisonTests.cs:23]: what does this look like before vs now in the test explorer in vs?

Comment (reddyashish) [reply src/VisualizationTests/HelixImageComparisonTests.cs:23]: Before:
Screenshot 2023-07-10 at 3 06 54 PM

After:
Screenshot 2023-07-10 at 3 10 41 PM

**Review by QilongTang — APPROVED** [review]

**Comment by mjkkirschner** [comment]
I'm curious if the tests will pass on this pr.

**Comment by reddyashish** [comment]
All the tests have passed. 
Also in this PR, adding a try-catch to handle invalid file path exception(https://jira.autodesk.com/browse/DYN-5962)

**Comment by reddyashish** [comment]
Failing with this message: Error: Failed to restore cache entry. Exiting as fail-on-cache-miss is set.

**Pull request merged**